src/client.cpp

```diff
@@ -1027,6 +1027,10 @@ void writePlayerPos(LocalPlayer *myplayer, ClientMap *clientMap, NetworkPacket *
 	u8 wanted_range  = MYMIN(255,
 			std::ceil(clientMap->getControl().wanted_range / MAP_BLOCKSIZE));
 
+	if (clientMap->getCameraFrontView()) {
+		yaw += 180 * 100;
```

Comment:
can you add a define for 180 to define what is this and use it in the multiple places ?